docs/en-US/component/date-picker.md

```diff
@@ -161,37 +161,39 @@ Note, date time locale (month name, first day of the week ...) are also configur
+| validate-event               | whether to trigger form validation                                                                                           | ^[boolean]                                                                                                                                                                  | true          |
+| disabled-date                | a function determining if a date is disabled with that date as its parameter. Should return a Boolean                        | ^[Function]`(data: Date) => boolean`                                                                                                                                        | —             |
+| shortcuts                    | an object array to set shortcut options                                                                                      | ^[object]`Array<{ text: string, value: Date \| Function }>`                                                                                                                 | []            |
+| cell-class-name              | set custom className                                                                                                         | ^[Function]`(data: Date) => string`                                                                                                                                         | —             |
+| teleported                   | whether date-picker dropdown is teleported to the body                                                                       | ^[boolean]                                                                                                                                                                  | true          |
+| empty-values ^(2.7.0)        | empty values of component, [see config-provider](/en-US/component/config-provider#empty-values-configurations)               | ^[array]                                                                                                                                                                    | —             |
+| value-on-clear ^(2.7.0)      | clear return value, [see config-provider](/en-US/component/config-provider#empty-values-configurations)                      | ^[string] / ^[number] / ^[boolean] / ^[Function]                                                                                                                            | —             |
+| fallback-placements ^(2.8.4) | list of possible positions for Tooltip [popper.js](https://popper.js.org/docs/v2/modifiers/flip/#fallbackplacements)         | ^[arrary]`Placement[]`                                                                                                                                                      | —             |
```

Comment:
Perhaps we can add a description of the type of `Placement` in the `Type Declarations`.

Reply:
Good advice!